frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamSettings/TeamSettings.tsx

```diff
@@ -190,15 +190,16 @@ const TeamSettings = ({ location, router }: ITeamSubnavProps) => {
       setPercentageHostsDropdownOptions(
         getCustomDropdownOptions(
           HOST_STATUS_WEBHOOK_HOST_PERCENTAGE_DROPDOWN_OPTIONS,
-          teamConfig?.webhook_settings.host_status_webhook.host_percentage ?? 1,
+          teamConfig?.webhook_settings?.host_status_webhook?.host_percentage ??
+            1,
```

Comment:
Jacob, What does 1, do here?

Reply:
Meaning, in the case BE returns null, this will be 1. What will the UI do with this 1?